src/Castle.Facilities.Logging/LoggingFacility.cs

```diff
@@ -83,6 +86,7 @@ public LoggingFacility(LoggerImplementation loggingApi) : this(loggingApi, null)
 		/// </summary>
 		/// <param name="loggingApi"> The LoggerImplementation that should be used </param>
 		/// <param name="configFile"> The configuration file that should be used by the chosen LoggerImplementation </param>
+		[Obsolete("A logger factory implementation type should be provided via LogUsing<T>(), this will be removed in the future.")]
```

Comment:
There are currently 131 Obsolete attributes in the code base. Can't we just delete this altogether? Assuming we kick the version up to v5.